src/ext/java/org/opentripplanner/ext/transmodelapi/TransmodelGraphQLPlanner.java

```diff
@@ -127,19 +134,21 @@ private RoutingRequest createRequest(DataFetchingEnvironment environment) {
         BicycleOptimizeType bicycleOptimizeType = environment.getArgument("bicycleOptimisationMethod");
 
         if (bicycleOptimizeType == BicycleOptimizeType.TRIANGLE) {
-            try {
-                RoutingRequest.assertTriangleParameters(
-                    request.bikeTriangleSafetyFactor,
-                    request.bikeTriangleTimeFactor,
-                    request.bikeTriangleSlopeFactor
-                );
-                // TODO These are not defined in the graphql schema
-                callWith.argument("triangle.safetyFactor", request::setBikeTriangleSafetyFactor);
-                callWith.argument("triangle.slopeFactor", request::setBikeTriangleSlopeFactor);
-                callWith.argument("triangle.timeFactor", request::setBikeTriangleTimeFactor);
-            } catch (ParameterException e) {
-                throw new RuntimeException(e);
-            }
+
+            // because we must use a final variable in the lambda we have to use this ugly crutch.
+            final AtomicReference<Double> safety = new AtomicReference<>((double) 0);
+            final AtomicReference<Double> slope = new AtomicReference<>((double) 0);
+            final AtomicReference<Double> time = new AtomicReference<>((double) 0);
+
+            callWith.argument("triangleFactors.safety", safety::set);
+            callWith.argument("triangleFactors.slope", slope::set);
+            callWith.argument("triangleFactors.time", time::set);
+
+            request.setTriangleNormalized(
+                    safety.get(),
+                    slope.get(),
+                    time.get()
+            );
```

Comment:
Same comment apply to this as the one in the Legacy API - it should be done in the same way.

Reply:
Done.